includes/admin/tools/export/class-batch-export-donors.php

```diff
@@ -167,6 +170,20 @@ public function get_data() {
 
 		$i = 0;
 
+		$donor_args = array(
+			'number' => 30,
+		);
+
+		// Filter Donors list based on specified timeframe
+		if ( ! empty( $this->data['donor_export_start_date'] ) || ! empty( $this->data['donor_export_end_date'] ) ) {
+
+			$donor_args['date'] = array(
+				'start'     => isset( $this->data['donor_export_start_date'] ) ? date( 'Y-n-d 00:00:00', strtotime( $this->data['donor_export_start_date'] ) ) : date( 'Y-n-d 23:59:59', mktime( 23, 59,59, 01, 01, 1970 ) ),
```

Comment:
@mehul0810 default value for date param if any of date empty:
1. start_date: `1970-1-01 00:00:00`
2. end date: `date(  'Y-n-d 23:59:59', current_time('timstamp')  )`

and use `empty` instead of `isset`